src/components/site/Images.md

```diff
@@ -0,0 +1,63 @@
+```jsx
+<div className="dg_image-content">
+  <img
+    src="//baltimorecountymd.gov/sebin/t/t/homepage-county-executive.jpg"
+    alt="County Executive Olzsewski delivers a speech."
+  />
+  <img
+    className="dg_image left"
+    src="//baltimorecountymd.gov/sebin/t/t/homepage-county-executive.jpg"
+    alt="County Executive Olzsewski delivers a speech."
+  />
+  <p>
+    Lorem Ipsum is simply dummy text of the printing and typesetting of Lorem
+    Ipsum.industry. Lorem Ipsum has been the industry's standard dummy text ever
+    since the 1500s, when an unknown printer took a galley of type and scrambled
+    it to make a type specimen book. It has survived not only five
+  </p>
+  <img
+    className="dg_image right"
+    src="//baltimorecountymd.gov/sebin/t/t/homepage-county-executive.jpg"
+    alt="County Executive Olzsewski delivers a speech."
+  />
+  <p>
+    Lorem Ipsum is simply dummy text of the printing and typesetting of Lorem
+    Ipsum.industry. Lorem Ipsum has been the industry's standard dummy text ever
+    since the 1500s, when an unknown printer took a galley of type and scrambled
+    it to make a type specimen book. It has survived not only five
+  </p>
+</div>
+```
+
+Html Snippet:
+
+```html
+<div class="dg_image-content">
```

Comment:
this div needs to change to `<div id="dg_main-content"><!-- stuff goes here --> </div>`. This denote the main content section of the application.

The css will have to change accordingly

Reply:
done